tests/threshold/threshold-rule-flow/test.yaml

```diff
@@ -0,0 +1,24 @@
+requires:
+  min-version: 8
+
+checks:
+  - filter:
+      count: 8
+      match:
+        event_type: alert
+        alert.signature_id: 1
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 2
+  - filter:
+      count: 2
+      match:
+        event_type: alert
+        alert.signature_id: 3
+  - filter:
+      count: 1
```

Comment:
Why is there only one alert ?
Could you add a comment about `threshold:type both` intention ?

Reply:
No, this is not the place to document features. Check the thresholding docs to learn how the thresholding mechanics work.

Reply:
The numbers still do not look obvious to me... Why did you pick 3 ?